Userland/Libraries/LibGUI/CommonActions.cpp

```diff
@@ -197,6 +198,22 @@ NonnullRefPtr<Action> make_rotate_counterclockwise_action(Function<void(Action&)
     return GUI::Action::create("Rotate &Counterclockwise", { Mod_Ctrl | Mod_Shift, Key_LessThan }, Gfx::Bitmap::try_load_from_file("/res/icons/16x16/edit-rotate-ccw.png"sv).release_value_but_fixme_should_propagate_errors(), move(callback), parent);
 }
 
+NonnullRefPtr<Action> make_search_action(Window* window)
+{
+    auto action = Action::create("&Search...", { Mod_Ctrl | Mod_Shift, Key_A }, MUST(Gfx::Bitmap::try_load_from_file("/res/icons/16x16/find.png"sv)), [=](auto&) {
```

Comment:
"Search" is not the right term for this, call it something like "command_palette_action" and "Commands..."

Reply:
:+1: